src/Page/Partners.elm

```diff
@@ -115,23 +115,29 @@ viewPartner partner =
 
 viewMap : List Data.PlaceCal.Partners.Partner -> Html msg
 viewMap partnerList =
+    let
+        allowOnlyPartnersWithLocation =
+            \partner ->
+                List.isEmpty partner.areasServed && (partner.maybeGeo /= Nothing)
```

Comment:
```suggestion
        partnersIsVenueWithLocation partner =
            List.isEmpty partner.areasServed && (partner.maybeGeo /= Nothing)
```

Reply:
This suggestion has two changes:

1. Renaming the function
2. Changing the function signature

Even if we disagree on (1), (2) is worth considering. It's _not_ a formatting change.